entrypoint.sh

```diff
@@ -12,6 +12,10 @@ BUILD_TAGS=""
 RESULTS_DIR=/results
 CEPH_CONF=/tmp/ceph/ceph.conf
 
+# env vars consumed by go code directly.
+# set defaults if they are currently unset in the environment
+: "${GO_CEPH_TEST_REQUIRE_MOUNT:=yes}"
```

Comment:
You also added GO_CEPH_TEST_MOUNT_DIR and GO_CEPH_TEST_MDS_NAME in the code. The least you should do is mention them here, possibly commented out.

Reply:
Sounds like a good idea.